src/Repositories/ExtensionRepository.php

```diff
@@ -136,8 +176,6 @@ public function createExtension(array $apiPackage)
 
         $this->refreshInstalledExtension($extension);
 
-        $this->flushCacheKey('flagrow.io.search.list');
```

Comment:
Can you add a second argument in createExtension `$flushCache = true` then set it to false when we parse results from the api? That would solve this..

A drawback of removing this line all together is an issue caused when we add an extension to the UX that wasn't received from Flagrow.io but is installed.